src/app/components/data-entry-grid/index.html

```diff
@@ -103,6 +103,9 @@ <h3>Guidelines</h3>
       <p>
         In addition to the <sky-code>@skyux/ag-grid</sky-code> package, you need to install the <a href="https://www.ag-grid.com/angular-grid/"><sky-code>ag-grid-angular</sky-code></a> and <a href="https://www.npmjs.com/package/ag-grid-community"><sky-code>ag-grid-community</sky-code></a> packages. After installation, you need to make any exports from <sky-code>SkyAgGridModule</sky-code> available to your SPA by adding the module to the <sky-code>exports</sky-code> section of your SPA's main module. For example, if you are using the SKY UX SDK, add <sky-code>SkyAgGridModule</sky-code> as an export in your <sky-code>app-sky.module.ts</sky-code> file. Next, in the <sky-code>app-extras.module.ts</sky-code> file, add <sky-code>AgGridModule.withComponents([])</sky-code> to the <sky-code>imports</sky-code> section and <sky-code>AgGridModule</sky-code> to the <sky-code>exports</sky-code> section. If you define your own cell editors or renderers, add them to the list passed to <a href="https://www.ag-grid.com/angular-more-details/#angular-components"><sky-code>withComponents()</sky-code></a>.
       </p>
+      <sky-alert alertType="info">
+        IF you use the data entry grid with <a href="https://www.ag-grid.com/documentation/angular/licensing/"></a>the enterprise version of ag-Grid instead of the free community version</a>, the data entry grid turns on all enterprise features by default.
```

Comment:
```suggestion
        If you use the data entry grid with <a href="https://www.ag-grid.com/documentation/angular/licensing/"></a>the enterprise version of ag-Grid instead of the free community version</a>, the data entry grid turns on all enterprise features by default.
```